e2e/test/scenarios/permissions/sandboxing/sandboxing-misconfiguration.cy.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { USER_GROUPS, WRITABLE_DB_ID } from "e2e/support/cypress_data";
+
+import {
+  assertResponseFailsClosed,
+  assignAttributeToUser,
+  configureSandboxPolicy,
+  getCardResponses,
+  gizmoViewer,
+  rowsShouldContainOnlyOneCategory,
+  signInAs,
+} from "./helpers/e2e-sandboxing-helpers";
+
+const { H } = cy;
+
+describe("admin > permissions > sandboxing > misconfiguration", () => {
+  before(() => {
```

Comment:
I think having a `before()` without a `beforeEach()` here is a trap. Currently there's only one test here, so it's fine, but if someone adds another test, they might not realize it's dependent on the first test. I would make a snapshot at the end of `before()` and then do another restore and signin in a `beforeEach` hook.